data/campaigns/Winds_of_Fate/units/elementals/Storm_Daemon.cfg

```diff
@@ -0,0 +1,172 @@
+#textdomain wesnoth-wof
+
+[unit_type]
+    id=Storm Daemon
+    name= _ "Storm Daemon"
+    race=elementals
```

Comment:
I dont think `race=elementals` or `race=elemental` exists for core units right now. Resolve.

Reply:
Elementals are `race=monster` with `{TRAIT_ELEMENTAL}`.

Reply:
For sure, the simplest thing to do here is put it into race monster. But since adding an elemental race and more elemental units (as per #5532) is on [the roadmap](https://wiki.wesnoth.org/1.17_Roadmap#1.17.20_(09/17/2023)) the other option is to add the elemental race now and move the existing Fire Guardian/Wraith, Mudcrawler, Jinni and Dust Devil into it, along with this new Storm Daemon.